exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/DatabaseConfig.kt

```diff
@@ -34,57 +34,57 @@ class DatabaseConfig private constructor(
          */
         var defaultFetchSize: Int? = null,
         /**
-         * Default transaction isolation level. If not specified database-specific level will be used
-         * Can be overridden on per-transaction level by specifying `transactionIsolation` parameter of `transaction` function
-         * Check [Database.getDefaultIsolationLevel] for the database defaults
+         * Default transaction isolation level. If not specified database-specific level will be used.
+         * Can be overridden on per-transaction level by specifying `transactionIsolation` parameter of `transaction` function.
+         * Check [Database.getDefaultIsolationLevel] for the database defaults.
          */
         var defaultIsolationLevel: Int = -1,
         /**
-         * How many retries will be made inside any `transaction` block if SQLException happens
-         * Can be overridden on per-transaction level by specifying `repetitionAttempts` parameter on call
-         * Default attempts are 3
+         * How many retries will be made inside any `transaction` block if SQLException happens.
+         * Can be overridden on per-transaction level by specifying `repetitionAttempts` property in `transaction` block.
+         * Default attempts are 3.
```

Comment:
All these small changes are just because I noticed that the KDocs in the IDE (when a property is hovered over) were showing up as 1 long multi-line blob of text with no full-stop. The other option would have been to use a line break (empty line) in between each sentence.